deploy-board/deploy_board/webapp/helpers/nimbusclient.py

```diff
@@ -41,6 +41,18 @@ def get_one_identifier(self, name, token=None):
         return self.handle_response(response)
 
     def create_one_identifier(self, data, token=None):
+        """
+        Create a Nimbus Identifier according to the input request data.
+        If the request data does not have all the information needed for creating a Nimbus identifier, this method will raise a Teletraan Exception.
+        """
+        requiredParams = ['projectName', 'env_name', 'stage_name']
+        for param in requiredParams:
+            if data.get(param) is None or len(data.get(param)) == 0:
+                log.error("Missing %s in the request data, cannot create a Nimbus identifier" % param)
+                if IS_PINTEREST:
```

Comment:
this logic will only raise exception when IS_PINTEREST.  maybe always raise exception only modify exception msg to wrap "contect ....." under IS_PINTEREST

Reply:
or better yet, wrap all nimbus integration under IS_PINTEREST in the future.

Reply:
thanks. i updated the code to always raise an exception.